### PR TITLE
feat: added new list images functionality

### LINKED ISSUE #21: [CAM 4-2] Nexus list process fct images
blocked by ospin-web-dev/hambda#977

### Description

Nexus needs a method that calls to the process fct image list endpoint.

---

### Deliverable(s)

- [ ] `nexus.process.functionality.image.list` *exists*
- [ ] once deployed, point out to mo the endpoint exists. with that, he can test end to end the process posting from the process

### PR DESCRIPTION
**Pull request by FelixMZ2018**
end to end testing is in https://github.com/ospin-web-dev/hambda/pull/1059

### CODE REVIEW

**Review by vdiezel — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "test-release": "npx semantic-release --dry-run",
     "test": "jest",
     "test-with-coverage": "jest --coverage",
+    "testWithCoverage": "jest --coverage",
```
Comment (DanielSeehausen) [inline package.json:24]: what's this sneaky guy for when we have `test-with-coverage`?

File: src/process/functionality/image/list.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const Amplify = require('aws-amplify')
+const serializeAxiosResponse = require('../../../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../../../utils/defaultReqOpts')
+
```
Comment (DanielSeehausen) [inline src/process/functionality/image/list.js:4]: if we don't document what the q params are, we may want to reference to where someone can find them

**Review by DanielSeehausen — APPROVED** [review]
lgtm! nothing blocking

only request is to help guide the user (or give them directly) the query parameters !

File: package.json (hunk 1 above)
Comment (FelixMZ2018) [reply package.json:24]: I think you might be better at remembering which repo uses the dashed commands and which one the snake case ones so this way no one has to change their habits

File: src/process/functionality/image/list.js (hunk 2 above)
Comment (FelixMZ2018) [reply src/process/functionality/image/list.js:4]: they should all be in the swagger docs which is linked in the top of the repo, or should we link directly to the endpoint docs for all functions ?

**Pull request merged**